internal/runnerinstall/k8s.go

```diff
@@ -101,22 +100,25 @@ func (i *K8sRunnerInstaller) Install(ctx context.Context, opts *InstallOpts) err
 
 	version := i.Config.Version
 	if version == "" {
-		version = helminstallutil.DefaultHelmChartVersion
+		tags, err := helminstallutil.GetLatestHelmChartVersion(ctx)
+		if err != nil {
+			opts.UI.Output("Error getting latest tag of Waypoint helm chart.", terminal.WithErrorStyle())
+			return err
+		}
+		version = *tags[0].Name
```

Comment:
I was just thinking "Oh we should only do this if the user doesn't provide an error so we don't fail out unnecessarily" but then I saw that's exactly what you did 🎉